docs/samples/Microsoft.ML.GenAI.Samples/Microsoft.ML.GenAI.Samples.csproj

```diff
@@ -16,10 +16,11 @@
   </ItemGroup>
 
   <ItemGroup>
-    <PackageReference Include="TorchSharp-cuda-windows" Version="0.102.5" Condition="$([MSBuild]::IsOSPlatform('Windows'))" />
+    <PackageReference Include="TorchSharp-cuda-windows" Version="0.105.0" Condition="$([MSBuild]::IsOSPlatform('Windows'))" />
     <PackageReference Include="Microsoft.SemanticKernel" Version="$(SemanticKernelVersion)" />
     <PackageReference Include="AutoGen.SourceGenerator" Version="$(AutoGenVersion)" />
     <PackageReference Include="Microsoft.Extensions.Logging.Console" Version="8.0.0" />
+    <PackageReference Include="LittleLittleCloud.TorchSharp.BitsAndBytes" Version="0.0.4" />
```

Comment:
Is this not something we can directly get into torchsharp itself?

Reply:
Probably not, `bitsandbytes` is not part of `libtorch`...

Reply:
Is the code in a repo owned by you? Or by Microsoft?

Reply:
And we are going to want the version in a central location since you use it in more than once place.

Reply:
> Is the code in a repo owned by you? Or by Microsoft?

The wrapping code is in a repo owned by myself. The cuda code is owned by huggingface I believe. Both are under MIT license.